api/src/paths/project/{projectId}/survey/{surveyId}/critters/{critterId}/telemetry.ts

```diff
@@ -0,0 +1,310 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { PROJECT_PERMISSION, SYSTEM_ROLE } from '../../../../../../../constants/roles';
+import { getDBConnection } from '../../../../../../../database/db';
+import { HTTP400 } from '../../../../../../../errors/http-error';
+import { authorizeRequestHandler } from '../../../../../../../request-handlers/security/authorization';
+import { BctwService } from '../../../../../../../services/bctw-service';
+import { ICritterbaseUser } from '../../../../../../../services/critterbase-service';
+import { SurveyCritterService } from '../../../../../../../services/survey-critter-service';
+import { getLogger } from '../../../../../../../utils/logger';
+
+const defaultLog = getLogger('paths/project/{projectId}/survey/{surveyId}/critters/{critterId}/telemetry');
+export const GET: Operation = [
+  authorizeRequestHandler((req) => {
+    return {
+      or: [
+        {
+          validProjectPermissions: [PROJECT_PERMISSION.COORDINATOR, PROJECT_PERMISSION.COLLABORATOR],
+          projectId: Number(req.params.projectId),
+          discriminator: 'ProjectPermission'
+        },
+        {
+          validSystemRoles: [SYSTEM_ROLE.DATA_ADMINISTRATOR],
+          discriminator: 'SystemRole'
+        }
+      ]
+    };
+  }),
+  getCritterTelemetry()
+];
+
+GET.apiDoc = {
+  description: 'Get telemetry points for a specific critter.',
+  tags: ['bctw'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'surveyId',
+      schema: {
+        type: 'integer'
+      },
+      required: true
+    },
+    {
+      in: 'path',
+      name: 'critterId',
+      schema: {
+        type: 'integer'
+      },
+      required: true
+    },
+    {
+      in: 'query',
+      name: 'startDate',
+      schema: {
+        type: 'string'
+      }
+    },
+    {
+      in: 'query',
+      name: 'endDate',
+      schema: {
+        type: 'string'
+      }
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Responds with count of rows created in SIMS DB Deployments.',
+      content: {
+        'application/json': {
+          schema: {
+            title: 'Telemetry response',
+            type: 'object',
+            properties: {
+              points: {
+                type: 'object',
+                properties: {
+                  type: {
+                    type: 'string'
+                  },
+                  features: {
```

Comment:
We have GeoJSON openapi definitions in `geoJson.ts` that we should use. Just to ensure the geojson types are consistent across all endpoints. 

The 2 Feature types don't define anything specific in the properties field, so you'd have to assign it manually in this file before adding it to the responses section here.